lib/src/models/seed_changes/seed_key_diffed.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:equatable/equatable.dart';
+
+import 'package:nekoton_repository/nekoton_repository.dart';
+
+/// [SeedKeyDiffed] is [SeedKey] (even master key=seed).
+/// This model allows tracking SeedKey adding/removing from [SeedList],
+/// see [SeedListDiffChange].
+class SeedKeyDiffed extends Equatable {
```

Comment:
Hey, it's really huge amount of copy and paste code isn't it? :)

Reply:
-